artifactory.py

```diff
@@ -1757,17 +1819,22 @@ def find_permission_target(self, name):
             return obj
         return None
 
-    def writeto(self, out, chunk_size=None):
+    def writeto(self, output, chunk_size=1024, progress_func=log_download_progress):
```

Comment:
I did not notice it, but we should keep our public interface the same. Here we changed `out=>output` without necessity. I'll revert it back to keep our interface back compatible

Reply:
When I was creating a function, writeto was not in production
It was added only in release 2 weeks ago